### DISCUSSION TITLE
grafana/ui: The curent date is not highlighted within the Date picker

### DISCUSSION THREAD

**Discussion by hawran** [grafana/grafana#46863] [Dashboards, Panels & Visualizations]
**What happened**:
Trying to set up a date/time range for a dashboard via the date picker.

**What you expected to happen**:
The current date is highlighted.

**How to reproduce it (as minimally and precisely as possible)**:
See above.

**Environment**: 
- Grafana version: 8.4.4
- User OS & Browser: Ubuntu, Firefox 98.0.1 (64-bit)

**Comment by hawran** [comment]
As *a picture is worth a thousand words*:
![220322-135313 grafana current-date](https://user-images.githubusercontent.com/25924718/159488367-1d0dcd89-c501-46d8-854b-642c7a71278e.png)
(the current date is `22/3/2022`)

**Comment by hawran** [comment]
Hi, why is this a discussion now?
I'm pretty sure I've reported a regression, the current date was highlighted in previous versions.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
